index.html

```diff
@@ -62,6 +63,13 @@ <h5 class="nav-category">
       <button type="button" id="button-tray" data-section="tray" class="nav-button">Put your app in the <em>tray</em></button>
     </div>
 
+    <div class="nav-item u-category-notifications">
```

Comment:
What do you think about putting this under the "Native User Interface" section instead of creating a new section?